pom.xml

```diff
@@ -14,6 +14,11 @@
   <name>Jenkins SSH Slaves plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/SSH+Slaves+plugin</url>
 
+  <properties>
+    <findbugs-maven-plugin.version>3.0.1</findbugs-maven-plugin.version>
```

Comment:
Well, it not so bad imho in comparison to redundant ancient pom project https://github.com/jenkinsci/pom